doc/orchestrator.rst

```diff
@@ -1,208 +1,685 @@
+    requires that the ``Model`` finish before the ``Experiment`` continues.
+    This is because we will request tensors from the ``Orchestrator`` that
+    are inputted by the ``Model`` we launched.
+
+Data Polling
+------------
+Next, check if the tensor exists in the standalone ``Orchestrator`` using ``Client.poll_tensor()``.
+This function queries for data in the ``Orchestrator``. The function requires the tensor name (`name`),
+how many milliseconds to wait in between queries (`poll_frequency_ms`),
+and the total number of times to query (`num_tries`). Check if the data exists in the ``Orchestrator`` by
+polling every 100 milliseconds until 10 attempts have completed:
 
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 38-41
 
-By default, SmartSim will pin the database to the first _N_ CPUs according to ``db_cpus``. By
-specifying the optional argument ``custom_pinning``, an alternative pinning can be specified
-by sending in a list of CPU ids (e.g [0,2,range(5,8)]). For optimal performance, most users
-will want to also modify the RunSettings for the model to pin their application to cores not
-occupied by the database.
+When you execute the driver script, the output will be as follows::
 
-.. warning::
+  23:45:46 system.host.com SmartSim[87400] INFO The tensor exists: True
 
-  Pinning is not supported on MacOS X. Setting ``custom_pinning`` to anything
-  other than ``None`` will raise a warning and the input will be ignored.
+Cleanup
+-------
+Finally, use the ``Experiment.stop()`` function to stop the ``Orchestrator`` instance. Print the
+workflow summary with ``Experiment.summary()``:
 
-.. note::
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 43-46
+
+When you run the experiment, the following output will appear::
+
+  |    | Name           | Entity-Type   | JobID       | RunID   | Time    | Status    | Returncode   |
+  |----|----------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 0  | model          | Model         | 1658679.3   | 0       | 1.3342  | Completed | 0            |
+  | 1  | orchestrator_0 | DBNode        | 1658679.2+2 | 0       | 42.8742 | Cancelled | 0            |
+
+.. _colocated_orch_doc:
+====================
+Colocated Deployment
+====================
+--------
+Overview
+--------
+During colocated ``Orchestrator`` deployment, a SmartSim ``Orchestrator`` (the database) runs on
+the ``Models`` compute node(s). Colocated ``Orchestrators`` can only be deployed as isolated instances
+on each compute node and cannot be clustered over multiple nodes. The ``Orchestrator`` on each application node is
+utilized by SmartRedis ``Clients`` on the same node. With a colocated ``Orchestrator``, latency is reduced
+in ML inference and TorchScript evaluation by eliminating off-node communication. A colocated ``Orchestrator``
```

Comment:
all interactions with the database occur on the same node, thus resulting in lower latency compared to the standard ``Orchestrator``